security/src/main/java/com/root/security/sample/SampleUsage.kt

```diff
@@ -0,0 +1,58 @@
+package com.root.security.sample
+
+import android.app.Activity
+import com.root.security.AdessoSecurityProvider
+import com.root.security.PublicKeyPinner
+import com.root.security.SocketProvider
+import com.root.security.TrustStore
+import com.root.security.utility.CertificateUtility
+import okhttp3.OkHttpClient
+
+/**
+ * @author haci
+ * @version 0.0.1
+ * @since 0.0.1
+ * Adesso Security Module.
+ * created on 21.06.2021
+ */
+class SampleUsage : Activity() {
+
+    fun blabla() {
+
+        val playSecurity = AdessoSecurityProvider.getDefaultSecurityProvider(this)
+        val trustStore = AdessoSecurityProvider.getTrustStore()
+        val socketProvider = AdessoSecurityProvider.getSocketProvider()
+        val publicKeyPinner = AdessoSecurityProvider.getOkHttpPublicKeyPinner()
+
+        playSecurity.update()
+        okHttpCertificatePinning(trustStore, socketProvider)
+        okHttpPublicKeyPinning(publicKeyPinner)
+    }
+
+    private fun okHttpPublicKeyPinning(publicKeyPinner: PublicKeyPinner) {
+        publicKeyPinner
+            .add(
```

Comment:
there is a real usage sample which we can use for here:  "https://api.themoviedb.org/3/" It also has certificate to use